internal/http/post.go

```diff
@@ -1,27 +1,25 @@
-package routes
+package http
 
 import (
 	"context"
 	"encoding/json"
+	"github.com/gorilla/mux"
 	"net/http"
 	"sublinks/sublinks-federation/internal/activitypub"
 	"sublinks/sublinks-federation/internal/lemmy"
-	"sublinks/sublinks-federation/internal/log"
-
-	"github.com/gorilla/mux"
 )
 
-func SetupPostRoutes(r *mux.Router) {
-	r.HandleFunc("/post/{postId}", getPostHandler).Methods("GET")
+func (s Server) SetupPostRoutes() {
+	s.HandleFunc("/post/{postId}", s.getPostHandler).Methods("GET")
```

Comment:
The same thing here as the logging.  `server.HandleFunc` just doesn't make as much sense. Maybe if it was `server.AddRoute`? When I see `server.HandleFunc` I think of adding a middleware, not a route